tests/dummy/app/resolver.js

```diff
@@ -0,0 +1,3 @@
+import Resolver from 'ember-resolver';
```

Comment:
it was created by ember-cli addon blueprint, not sure if this file was missed intentionally